app/assets/stylesheets/osem-navbar.css.scss

```diff
@@ -0,0 +1,42 @@
+.btn_sign_in {
+    background-color: #24a724;
+    color: white;
+    font-weight: bold;
+    &:hover {
+        color: white;
+        background-color: rgba(41, 147, 11, 1);
+    }
+}
+
+.nav-modify {
+    border: none;
+}
+
+.navbar-collapse.in {
+    overflow: hidden;
+    max-height: none;
+}
+
+.nav-ham-btn .icon-bar {
+    background-color: white !important;
```

Comment:
The usage of `!important` is a big red flag that there's something wrong in the CSS structure.

While the end result of this PR is fine, I think we can still improve the CSS.

How about this?

Instead of making up a bunch of new classes with cryptic names (`.nav-ham-btn`, `.nav-mdfy-collapse`, `.drop-tgl`, etc.), I propose that you create a single CSS class called `.navbar-osem`, and apply it to the navbar at https://github.com/openSUSE/osem/pull/1285/files#diff-09ec3c6a44f3c765b3df29ff1e60d10eR1.

Then, if you need to override any of the default navbar rules, you can do something like:

```scss
.navbar-osem {
  .navbar-collapse { // This class is already in the HAML!
    // ...
  }
  .dropdown-toggle { // And this one!
    // ...
  }
  .dropdown-menu { // Also this one :)
    // ...
  }
  // Catch the flow? :)
}
```

All you need to create is one single new class and apply it to the navbar. How about it?

Also, let's drop all usages of `!important`. We don't need it. Have a look [at this blog post](http://zurb.com/university/lessons/what-specifically-is-css-specificity) to get a better understanding of CSS specificity, which will help you get rid of all those `!important`s :)